features/unmarshal/unmarshal.go

```diff
@@ -25,12 +25,17 @@ func init() {
 	generator.RegisterFeature("unmarshal_unsafe", func(gen *generator.GeneratedFile) generator.FeatureGenerator {
 		return &unmarshal{GeneratedFile: gen, unsafe: true}
 	})
+
+	generator.RegisterFeature("unmarshal_unique", func(gen *generator.GeneratedFile) generator.FeatureGenerator {
+		return &unmarshal{GeneratedFile: gen, unique: true}
+	})
```

Comment:
I'm afraid this doesn't feel generally useful enough to merge into the library. I can't think of a single use case where I'd want _all_ the strings in a ProtoBuf message to be interned. It feels like a DDOS in the making!

`vtprotobuf` already supports having optional features per-field (see https://github.com/planetscale/vtprotobuf/blob/main/include/github.com/planetscale/vtprotobuf/vtproto/ext.proto), so I would very much prefer if this interning functionality was scoped per field instead to a whole message. Thank you!